docs/pages/preview/api-routes.mdx

```diff
@@ -364,6 +365,137 @@ If you're using any environment variables or **.env** files, add them to Netlify
 
 </Step>
 
+### Vercel
+
+> **warning** This is experimental and subject to breaking changes. We have no continuous tests against this configuration.
+
+<Step label="1">
+
+Create a server entry file. All requests will be delegated through this middleware. The exact file location is important.
+
+```js api/index.js
+const { createRequestHandler } = require('@expo/server/adapter/vercel');
+
+module.exports = createRequestHandler({
+  /* @info Points to the root `dist/` (output) folder */
+  build: require('path').join(__dirname, '../../dist'),
+  /* @end */
+  mode: process.env.NODE_ENV,
+});
+```
+
+</Step>
+
+<Step label="2">
+
+Create a Vercel configuration file at the root of your project to redirect all requests to the server function.
+
+<Tabs>
+<Tab label="vercel.json v2">
+
+```json vercel.json
+{
+  "version": 2,
+  "outputDirectory": "dist",
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@vercel/static-build",
+      "config": {
+        "distDir": "dist"
+      }
+    },
+    {
+      "src": "api/index.js",
+      "use": "@vercel/node",
+      "config": {
+        "includeFiles": ["dist/**"]
+      }
+    }
+  ],
+  "routes": [
+    {
+      "src": "/_expo/functions/.*",
+      "dest": "/404",
+      "status": 404
+    },
+    {
+      "handle": "filesystem"
+    },
+    {
+      "src": "/(.*)",
+      "dest": "/api/index.js"
+    }
+  ]
+}
+```
+
+The **legacy** version of the `vercel.json` needs a `@vercel/static-build` runtime to serve your assets from the `dist/` output folder.
```

Comment:
```suggestion
The **legacy** version of the **vercel.json** needs a `@vercel/static-build` runtime to serve your assets from the **dist** output folder.
```

To maintain consistency in our docs, we prefer bold file/directory names as [per our writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text).